### ISSUE TITLE
Text stays in English and not being translated

### ISSUE THREAD

**Issue by mikeallbutt** [DiscipleTools/disciple-tools-theme#1174]
Here are some places where text in the code uses English and not available to be translated - but should be.

The last tile of the  /settings page

![Screen Shot 2021-02-02 at 17 33 27 ](https://user-images.githubusercontent.com/43966676/106676754-17d3b580-657d-11eb-8764-ff3e2cad8e31.png)

**Comment by micahmills** [comment]
If you find more untranslated strings please add them to this issue so we can have them all in one place.

**Comment by NathTech** [comment]
Hi, I'm looking for a simple PR to get started with understanding the codebase and getting it set up locally etc. would it be ok if I took this one on? I'll hunt for other non-translated strings as well.

**Comment by micahmills** [comment]
Yes please! Let us know if you have any questions or need any help.

**Comment by corsacca** [comment]
Hey, I've actually started on this on and forgot to close this issue.
https://github.com/DiscipleTools/disciple-tools-theme/commit/733d40679609e9cfa546c6d32f6706e42dd2ffa1

Push notifications is added from the mobile app plugin, so will still not be translated.

**Comment by corsacca** [comment]
@NathTech how does this one look? https://github.com/DiscipleTools/disciple-tools-theme/issues/951

**Comment by mikeallbutt** [comment]
#1334 has some new places where the text stays in English. (submenus and Plugin Headings).
